packages/algob/src/lib/algo-operator.ts

```diff
@@ -145,7 +145,7 @@ export class AlgoOperatorImpl implements AlgoOperator {
     account: rtypes.Account, params: algosdk.SuggestedParams,
     flags: rtypes.TxParams
   ): Promise<void> {
-    console.log(`ASA ${account.name} opt-in for ASA ${asaName}`);
+    console.log(`ASA ${String(account.name)} opt-in for ASA ${String(asaName)}`);
```

Comment:
why we need to cast to string? Isn't this already a string?

Reply:
because of yarn link while checking with github actions, in local system it doesn't give error
but with actions it does, i searched about why this is happening, got answer maybe tslint is missing some edge cases or these values maybe undefined at some point.